js/ui/iconGridLayout.js

```diff
@@ -104,7 +104,7 @@ const IconGridLayout = new Lang.Class({
     },
 
     _loadConfigJsonString: function(dir, base) {
-        let jsonString = null;
+        let jsonString = '{}';
```

Comment:
Actually, this will break things, because the code in _mergeJsonStrings is relying on this to be null for the missing data (as there are two possible sources of this information -- one part of the core OS, and one provided by the image builder).  Instead, I think the correct change is in _mergeJsonStrings, something along the lines of the following...
From:
```
        if (override) {
            baseNode = JSON.parse(override);
        } else {
            baseNode = JSON.parse(base);
        }
```
To:
```
        if (override) {
            baseNode = JSON.parse(override);
        } else if (base) {
            baseNode = JSON.parse(base);
        } else {
            baseNode = JSON.parse('{}');
        }
```